pkg/xds/bootstrap/generator.go

```diff
@@ -183,7 +183,7 @@ func (b *bootstrapGenerator) generateFor(proxyId core_xds.ProxyId, dataplane *co
 		}
 		certBytes = base64.StdEncoding.EncodeToString(cert)
 	}
-	accessLogPipe := fmt.Sprintf("/tmp/kuma-access-logs-%s-%s.sock", request.Name, request.Mesh)
+	accessLogPipe := fmt.Sprintf("/tmp/%s-%s.sock", request.Name, request.Mesh)
```

Comment:
Maybe it's better to create a function like: 
```go
func AccessLogPipeName(name, mesh string) string {
    return fmt.Sprintf("/tmp/%s-%s.sock", name, mesh)
}
```
and use it from both places? That'd allow avoiding bugs like this one, what do you think?